bundle-workflow/python/signing_workflow/signer.py

```diff
@@ -2,22 +2,27 @@
 
 # Copyright OpenSearch Contributors.
 # SPDX-License-Identifier: Apache-2.0
-import sys 
+import sys
+import os
 
 sys.path.insert(0,"../git")
 from git.git_repository import GitRepository
 
-
 '''
 This class is responsible for signing an artifact using the OpenSearch-signer-client and verifying its signature.
 The signed artifacts will be found in the same location as the original artifacts.
 '''
 class Signer:
     def __init__(self):
-        self.git_repo = GitRepository("https://github.com/opensearch-project/opensearch-signer-client.git", "HEAD")
+        self.git_repo = GitRepository(self.get_repo_url(), "HEAD")
         self.git_repo.execute("./bootstrap", subdirname = "src")
         self.git_repo.execute("rm config.cfg", subdirname = "src")
 
+    def get_repo_url(self):
+        if "GITHUB_TOKEN" in os.environ:
+            return "https://${GITHUB_TOKEN}@github.com/opensearch-project/opensearch-signer-client.git"
```

Comment:
Curious why the signer client is closed source?

Reply:
This is in the process of being opened up - [issue](https://github.com/opensearch-project/opensearch-signer-client/issues/5).